tests/test_parameters.py

```diff
@@ -43,17 +43,29 @@ def test_ParameterClasses():
     # assert emp.En1 == 1.0
 
     # Test MagneticMultipoleParameters
-    mmp = MagneticMultipoleParameters(Bn1=1.0, Bs1=0.5)
+    mmp = MagneticMultipoleParameters(tilt1=1.2, Bn1=1.0, Bs1=0.5)
     assert mmp.Bn1 == 1.0
     assert mmp.Bs1 == 0.5
```

Comment:
Should we check `tilt1` as well, since we're checking `Bn1` and `Bs1`? 

I think these kinds of checks are trivial, but after #33 we're doing it everywhere, so maybe we can do it consistently, i.e., for at least all parameters that we initialize explicitly:
```diff
-    mmp = MagneticMultipoleParameters(Bn1=1.0, Bs1=0.5)
+    mmp = MagneticMultipoleParameters(tilt1=1.2, Bn1=1.0, Bs1=0.5)
+    assert mmp.tilt1 == 1.2
     assert mmp.Bn1 == 1.0
     assert mmp.Bs1 == 0.5
```